iOS/ViewControllers/Courses/PeerAssessmentHintViewController.swift

```diff
@@ -97,6 +104,26 @@ class PeerAssessmentHintViewController: UIViewController {
         let number = NSNumber(value: self.courseItem.maxPoints)
         self.pointsLabel.text = Self.pointsFormatter.string(from: number).flatMap { String.localizedStringWithFormat(format, $0) }
 
+        // Set time effort label
+        let roundedTimeEffort = ceil(TimeInterval(self.courseItem.timeEffort) / 60) * 60 // round up to full minutes
+        self.timeEffortLabel.text = Self.timeEffortFormatter.string(from: roundedTimeEffort)
+        self.timeEffortView.isHidden = self.courseItem.timeEffort == 0
+
+        // Set deadline label
+        self.deadlineLabel.text = self.courseItem.deadline.map(Self.dateFormatter.string(from:))
+        self.deadlineDateView.isHidden = self.courseItem.deadline == nil
+
+        self.peerAssessmentInfoView.isHidden = false
+        self.peerAssessmentTypeView.isHidden = true
```

Comment:
I think this line should rather be placed in the else closure of the following guard statement, right?

Reply:
Actually not, because the peer assessment type is linked to the content of the peer assessment course item. Though, we can't display it at this point and it has to be hidden, unlike the other items in the peerAssessmentInfoView, which are all linked to the course item itself.